package.json

```diff
@@ -35,6 +36,7 @@
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-node": "^8.0.0",
     "eslint-plugin-promise": "^4.0.1",
-    "eslint-plugin-standard": "^4.0.0"
+    "eslint-plugin-standard": "^4.0.0",
+    "graphql-cli": "^3.0.3"
```

Comment:
I installed `npm install -g graphql-cli` refer to [this repo](https://github.com/graphql-cli/graphql-cli).
because of `graphql get-schema -p prisma` phrase in `prisma.yml`

Reply:
Yes but I don't understand why this is installed locally.

Reply:
Ah, sorry. I didn't install `graphql-cli` globally at my notebook. 

There is not `graphql-cli` in `package.json` at first time I forked, so I think you already installed `graphql-cli` globally in your environment. that's why I put in **devDependencies** and I think It wouldn't be used with coding.
and because I am newbie in graphql... plz let me know it is not right thing.

Reply:
So you've explained that we need `graphql-cli` in `prisma.yml` where in`post-deploy` where we `graphql get-schema -p prisma`. Thanks for explaination.